src/app/view/paperView.js

```diff
@@ -451,7 +451,8 @@ export default class PaperView {
     removeDevice() {
         if (this.paperDevice) this.paperDevice.remove();
         this.paperDevice = null;
-        this.featureRegistry = new Map();
+        //TODO: Figure out how to handle featureRegistry
+        //this.featureRegistry = new Map();
```

Comment:
Whats going on here ?

Reply:
featureRegistry is used to "remember" what feature belongs on what layer. It's used for some of the API I built for Ron. This comment is here because, I want to empty that map if the device is deleted, but, when importing a design, the map would get erased here, so I have that commented out.

Reply:
Would it break the implementation if @zyrrron updates the codebase ?